tests/nl/light_HassLightSet.yaml

```diff
@@ -20,6 +21,7 @@ tests:
       - "Felheid slaapkamer 75%"
       - "Doe slaapkamerverlichting naar 75%"
       - "Slaapkamerhelderheid 75%"
+      - "Zal je de slaapkamerverlichting naar 50% willen doen?"
```

Comment:
For this test the brightness is set to 75%, you are using 50% which is causing the test to fail